ui/src/checks/actions/thunks.ts

```diff
@@ -66,8 +66,11 @@ export const getChecks = () => async (
   getState: GetState
 ) => {
   try {
-    dispatch(setChecks(RemoteDataState.Loading))
-    const {id: orgID} = getOrg(getState())
+    const state = getState()
+    if (getStatus(state, ResourceType.Checks) === RemoteDataState.NotStarted) {
```

Comment:
what does this pattern mean?

Reply:
this is how I solved the getResources bug we saw last week. Now requests for resources that have already been loaded once do not cause an unmount, because they do not set RemoteDataState to Loaded, unless it's the first time.